src/app/public/src/modules/clipboard/clipboard.component.ts

```diff
@@ -15,33 +17,4 @@ export class StacheCopyToClipboardComponent implements OnInit {
 
   @Input()
   public buttonClickedText: string;
-
-  public buttonActive: boolean = false;
-  public enabled: boolean = false;
```

Comment:
Should you keep `enabled`? Everything that's public on the component is technically a breaking change when you remove it.

Reply:
It's no longer used on the component and wasn't an input.  It was used for a show hide on the button if the copy option was not available, with the recent update to use the pollyfill it's always available on all devices.